unity-ggjj/Assets/Scripts/NarrativeScript/NarrativeScriptPlayer/NarrativeScriptPlayerComponent.cs

```diff
@@ -45,16 +45,20 @@ public void TryPressWitness()
     }
 
     /// <summary>
-    /// Loads a narrative script, ending the current narrative script
-    /// and continuing the beginning of the loaded script
+    /// Loads a narrative script using a given narrative script name,
+    /// ending the current narrative script and
+    /// continuing the beginning of the loaded script
```

Comment:
Nit: "Continuing the beginning" reads a bit weird (since it's the start, it can't really continue), though I understand what the comment is trying to communicate. Would this work as an alternative?
```suggestion
    /// starting the newly loaded script
```

Reply:
It's supposed to be "continuing _at_ the beginning". Does it make more sense without the missing word?